api/utils.py

```diff
@@ -1,9 +1,28 @@
 from flask import current_app
 from authlib.jose import jwt
+import requests
 
 
-def get_payload(token):
+def get_payload(token: str) -> dict:
     """Decode data form jwt"""
     return dict(jwt.decode(token, current_app.config.get('SECRET_KEY')))
 
 
+def get_apod_info(url: str, date: str, hd: bool) -> dict:
+    """
+    Return link to picture of the day and picture's description.
+
+    url: NASA "APOD"-api's endpoint;
+    date: date for day's picture;
+    hd: True - high definition, False - lowe definition;
+
+    return - dict, where
+    explanation: picture's description
+    link: link for downloading.
+    """
+    res = requests.get(url, params={'date': date}).json()
+    link = res['hdurl'] if hd else res['url']
+    return {
+        'explanation': res['explanation'],
+        'link': link
+    }
```

Comment:
Why you are return only link, we agreed that we will return a picture?
Will you implement this in next task mb?